pages/glossary/clearly-labeled-location.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Clearly labeled location
+key: clearly-labeled-location
+unambiguous: true
+objective: false
+---
+
+Secondary information and alternative controls or functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
+
+The location of a target is said to be **"clearly labeled"** when the target can be found by activating [user interface components][] whose purpose, indicated by their [label], leads the user to find the target or to another [page][web page] or page state from which this action can be repeated until the target is found.
```

Comment:
I have to point out that "user interface component" is ambiguous (hence the whole "instrument" thingie), making this definition ambiguous.
I think that (WCAG) "label" is also ambiguous.

Reply:
Replaced "UI component" with "instrument".
Would like to hear the opinion of others regarding "label". Agree that it is not objective, but I don't find it ambiguous.